keyboards/mechlovin/olly/jf/rev2/keymaps/default_iso/keymap.c

```diff
@@ -0,0 +1,56 @@
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,             _______,    _______, _______, _______, _______,
+        _______, _______, _______,                            _______,                            _______, _______, _______,             _______, _______, _______,    _______, _______, _______
+    ),
+
+    [2] = LAYOUT_iso(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,    _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______,    _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,    _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,             _______,    _______, _______, _______, _______,
+        _______, _______, _______,                            _______,                            _______, _______, _______,             _______, _______, _______,    _______, _______, _______
+    ),
+
+    [3] = LAYOUT_iso(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,    _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______,    _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,    _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,             _______,    _______, _______, _______, _______,
+        _______, _______, _______,                            _______,                            _______, _______, _______,             _______, _______, _______,    _______, _______, _______
+    ),
+
```

Comment:
1. removed non-functional layers
2. no means stated to access layers 1-3
```suggestion
    )
```